src/Microsoft.Performance.Toolkit.Engine/Engine.cs

```diff
@@ -531,7 +530,7 @@ public void EnableTable(TableDescriptor descriptor)
             ITableExtensionReference tableReference = null;
             IEnumerable<ProcessingSourceExecutor> executorsWithTable = null;
 
-            if (!descriptor.IsInternalTable && this.Extensions.TablesById.TryGetValue(descriptor.Guid, out tableReference))
+            if (this.Extensions.TablesById.TryGetValue(descriptor.Guid, out tableReference))
```

Comment:
>(this.Extensions.TablesById.TryGetValue(descriptor.Guid, out tableReference)) [](http://example.com/codeflow?start=15&length=77)

should we double check that the descriptor has required cookers?

Reply:
These are pulled from the data extension repo, so they should be extensions. But I can add a debug check